.gitlab/deps_build.yml

```diff
@@ -29,18 +42,28 @@
       -DLLVM_ENABLE_BINDINGS=OFF \
       -DLLVM_PARALLEL_COMPILE_JOBS=4 \
       -DLLVM_PARALLEL_LINK_JOBS=4 \
+      -DLLVM_ENABLE_UNWIND_TABLES=OFF \
       ../llvm
     - cmake --build . --target install
-    - cd $DATADOG_AGENT_EMBEDDED_PATH
-    - rm -rf bin share libexec lib/clang lib/cmake lib/*.so*
+    # fail if references to glibc >= 2.18
+    - objdump -p $DATADOG_AGENT_EMBEDDED_PATH/bin/clang-11 | egrep 'GLIBC_2\.(1[8-9]|[2-9][0-9])' && exit 1
+    - objdump -p $DATADOG_AGENT_EMBEDDED_PATH/bin/llc | egrep 'GLIBC_2\.(1[8-9]|[2-9][0-9])' && exit 1
+    # fail if clang or llc targets anything other than bpf
+    - $DATADOG_AGENT_EMBEDDED_PATH/bin/clang-11 -print-targets > clang_targets.txt
+    - diff $SRC_PATH/pkg/ebpf/c/expected_targets.txt clang_targets.txt
+    - $DATADOG_AGENT_EMBEDDED_PATH/bin/llc --version | tail -n +7 > llc_targets.txt # remove first 6 lines of llc version output; we only care about the registered targets
+    - diff $SRC_PATH/pkg/ebpf/c/expected_targets.txt llc_targets.txt
+    - $S3_CP_CMD $DATADOG_AGENT_EMBEDDED_PATH/bin/clang-11 $S3_PERMANENT_ARTIFACTS_URI/clang-11.0.1.${ARCH}
```

Comment:
It has been requested that we name these binaries `clang-bpf` and `llc-bpf`, so the intent is a little clearer to our customers. Can you make that change?